frontend/src/types/button.ts

```diff
@@ -36,5 +36,11 @@ export type ButtonSize = (typeof buttonSizes)[number]
 export const buttonTypes = ["button", "submit", "reset"] as const
 export type ButtonType = (typeof buttonTypes)[number]
 
-export const buttonConnections = ["start", "end", "none", "all"] as const
+export const buttonConnections = [
```

Comment:
Technically the `buttonConnections` array should only contain 4 entries (one for each edge) and the `connections` prop on the `VButton` component should accept an array of connections. That would also remove the need for elements like `"none"` and `"all"`.

It can be a complex change so I won't block on it, especially because this works.

Reply:
Having an array for `buttonConnections` would be much cleaner! I'll see if I can implement this.